CONTRIBUTING.md

```diff
@@ -1,20 +1,49 @@
 How to contrubute to SunPy
 --------------------------
 
+Hello, and thanks for being interested in contributing to SunPy!
+
 SunPy is an open project that enourages everyone to contribute in any way possible.
-This document descibes how to contribute code and documentation to the main SunPy 
-repository, if you wish to make a different contribution please contact the 
-[mailing list](https://groups.google.com/forum/#!forum/sunpy).
+This document mainly focuses on code or documentation contributions, however
```

Comment:
Although this document mainly focuses on how to make contributions to SunPy code and documentation, there are many other ways to get involved with the SunPy community.  A good place to start is by saying hello on  the [mailinglist]() or [IRC]().